devicetypes/fibargroup/fibaro-wall-plug-us-zw5.src/fibaro-wall-plug-us-zw5.groovy

```diff
@@ -0,0 +1,496 @@
+/**
+ *  Fibaro Wall Plug ZW5
+ */
+metadata {
+    definition (name: "Fibaro Wall Plug US ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Switch"
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Configuration"
+        capability "Health Check"

```

Comment:
When using the "Health Check" capability it is strongly encouraged to add a "ping()" method. Device Health will call this method 1 minute before marking the device offline to test that we can communicate.